android_arm64-v8a.sh

```diff
@@ -87,13 +93,12 @@ PARSERS="\
 
 function build_arm64
 {
-    # no-missing-prototypes because of a compile error seemingly unique to aarch64.
 ./configure --logfile=conflog.txt --target-os=linux \
     --prefix=./android/arm64 \
     --arch=aarch64 \
     ${GENERAL} \
     --sysroot=$NDK/sysroot \
-    --extra-cflags=" --target=aarch64-none-linux-android21 -no-canonical-prefixes -fdata-sections -ffunction-sections -fno-limit-debug-info -funwind-tables -fPIC -O2 -DANDROID -DANDROID_PLATFORM=$BUILD_ANDROID_PLATFORM -Dipv6mr_interface=ipv6mr_ifindex -fasm -fno-short-enums -fno-strict-aliasing -Wno-missing-prototypes" \
+    --extra-cflags="$EXTRA_CC --target=aarch64-none-linux-android21 -no-canonical-prefixes -fdata-sections -ffunction-sections -fno-limit-debug-info -funwind-tables -g -O2 -DANDROID -DANDROID_PLATFORM=$BUILD_ANDROID_PLATFORM -Dipv6mr_interface=ipv6mr_ifindex -fasm -fno-short-enums -fno-strict-aliasing" \
```

Comment:
Intentional to leave `-g` here?  Also, last time I tried the latest NDK, there were issues related to PIC.  This stuff:

```
ld: error: relocation R_AARCH64_ADR_PREL_PG_HI21 cannot be used against symbol ff_h264_idct_dc_add_neon; recompile with -fPIC
ld: error: can't create dynamic relocation R_AARCH64_ADD_ABS_LO12_NC against symbol: ff_h264_idct_dc_add_neon in readonly segment; recompile object files with -fPIC or pass '-Wl,-z,notext' to allow text relocations in the output
```

Not sure if this is just old binaries or something, though.  I wonder if we need -pic or something else?

-[Unknown]

Reply:
pic is enabled with --enable-pic in the GENERAL settings above. -g might not be neeed, I just wanted as much stack trace info as I can get...